### ISSUE TITLE
rtk inspect throws Encountered a thing that does not look like a report!

### ISSUE THREAD

**Issue by bertyhell** [IBM/report-toolkit#62]
The report was generated using this file:
```javascript
// Define a custom type
function Type() {
  this.name = 'foo'
  this.id = 128
  this.account = 9845432470
}

// A global array, only to leak memory
const l = []

// Create half a million objects and fill those into the array
for (var i = 0; i < 500000; i++) {
  const r = new Type()
  l.push(r)
}

// Generate a report
process.report.writeReport()
```

The command to run the file and generate the report:
```
node --experimental-report --max-old-space-size=20 inspect.js
```

The report content:
```json

{
  "header": {
    "reportVersion": 1,
    "event": "JavaScript API",
    "trigger": "API",
    "filename": "report.20191113.151429.7984.0.001.json",
    "dumpEventTime": "2019-11-13T15:14:29Z",
    "dumpEventTimeStamp": "1573658069704",
    "processId": 7984,
    "cwd": "C:\\Users\\bert\\Documents\\studiohyperdrive\\repos\\nodejs-conf-report\\rtk",
    "commandLine": [
      "node",
      "--experimental-report",
      "--max-old-space-size=20",
      "inspect.js"
    ],
    "nodejsVersion": "v12.13.0",
    "wordSize": 64,
    "arch": "x64",
    "platform": "win32",
    "componentVersions": {
      "node": "12.13.0",
      "v8": "7.7.299.13-node.12",
      "uv": "1.32.0",
      "zlib": "1.2.11",
      "brotli": "1.0.7",
      "ares": "1.15.0",
      "modules": "72",
      "nghttp2": "1.39.2",
      "napi": "5",
      "llhttp": "1.1.4",
      "http_parser": "2.8.0",
      "openssl": "1.1.1d",
      "cldr": "35.1",
      "icu": "64.2",
      "tz": "2019a",
      "unicode": "12.1"
    },
    "release": {
      "name": "node",
      "lts": "Erbium",
      "headersUrl": "https://nodejs.org/download/release/v12.13.0/node-v12.13.0-headers.tar.gz",
      "sourceUrl": "https://nodejs.org/download/release/v12.13.0/node-v12.13.0.tar.gz",
      "libUrl": "https://nodejs.org/download/release/v12.13.0/win-x64/node.lib"
    },
    "osName": "Windows_NT",
    "osRelease": "10.0.18362",
    "osVersion": "Windows 10 Pro",
    "osMachine": "x86_64",
    "cpus": [
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 1049281,
        "nice": 0,
        "sys": 1062984,
        "idle": 21342500,
        "irq": 432484
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 450734,
        "nice": 0,
        "sys": 245500,
        "idle": 22758109,
        "irq": 33281
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 1069593,
        "nice": 0,
        "sys": 556734,
        "idle": 21828000,
        "irq": 9531
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 608500,
        "nice": 0,
        "sys": 246000,
        "idle": 22599812,
        "irq": 6015
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 739937,
        "nice": 0,
        "sys": 377375,
        "idle": 22337015,
        "irq": 8109
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 448718,
        "nice": 0,
        "sys": 313687,
        "idle": 22691921,
        "irq": 8203
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 801281,
        "nice": 0,
        "sys": 468171,
        "idle": 22184875,
        "irq": 10312
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 398593,
        "nice": 0,
        "sys": 275859,
        "idle": 22779875,
        "irq": 8609
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 944703,
        "nice": 0,
        "sys": 390609,
        "idle": 22119015,
        "irq": 8796
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 407453,
        "nice": 0,
        "sys": 227062,
        "idle": 22819812,
        "irq": 7062
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 611015,
        "nice": 0,
        "sys": 557953,
        "idle": 22285359,
        "irq": 9609
      },
      {
        "model": "Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz",
        "speed": 2208,
        "user": 369937,
        "nice": 0,
        "sys": 206390,
        "idle": 22877984,
        "irq": 17875
      }
    ],
    "networkInterfaces": [
      {
        "name": "vEthernet (DockerNAT)",
        "internal": false,
        "mac": "00:15:5d:01:36:1c",
        "address": "fe80::ad3e:4665:9c73:a5a4",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 6
      },
      {
        "name": "vEthernet (DockerNAT)",
        "internal": false,
        "mac": "00:15:5d:01:36:1c",
        "address": "10.0.75.1",
        "netmask": "255.255.255.240",
        "family": "IPv4"
      },
      {
        "name": "Wi-Fi",
        "internal": false,
        "mac": "9c:b6:d0:c0:8e:69",
        "address": "fe80::d897:d1fe:2b0f:7a1",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 15
      },
      {
        "name": "Wi-Fi",
        "internal": false,
        "mac": "9c:b6:d0:c0:8e:69",
        "address": "10.20.2.181",
        "netmask": "255.255.248.0",
        "family": "IPv4"
      },
      {
        "name": "Loopback Pseudo-Interface 1",
        "internal": true,
        "mac": "00:00:00:00:00:00",
        "address": "::1",
        "netmask": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff",
        "family": "IPv6",
        "scopeid": 0
      },
      {
        "name": "Loopback Pseudo-Interface 1",
        "internal": true,
        "mac": "00:00:00:00:00:00",
        "address": "127.0.0.1",
        "netmask": "255.0.0.0",
        "family": "IPv4"
      },
      {
        "name": "vEthernet (Default Switch)",
        "internal": false,
        "mac": "00:15:5d:3a:47:5a",
        "address": "fe80::3d5a:bb55:308d:8d01",
        "netmask": "ffff:ffff:ffff:ffff::",
        "family": "IPv6",
        "scopeid": 30
      },
      {
        "name": "vEthernet (Default Switch)",
        "internal": false,
        "mac": "00:15:5d:3a:47:5a",
        "address": "172.17.147.1",
        "netmask": "255.255.255.240",
        "family": "IPv4"
      }
    ],
    "host": "DESKTOP-K9VNH6V"
  },
  "javascriptStack": {
    "message": "Error [ERR_SYNTHETIC]: JavaScript Callstack",
    "stack": [
      "at Object.writeReport (internal/process/report.js:17:13)",
      "at Object.<anonymous> (C:\\Users\\bert\\Documents\\studiohyperdrive\\repos\\nodejs-conf-report\\rtk\\inspect.js:30:16)",
      "at Module._compile (internal/modules/cjs/loader.js:956:30)",
      "at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)",
      "at Module.load (internal/modules/cjs/loader.js:812:32)",
      "at Function.Module._load (internal/modules/cjs/loader.js:724:14)",
      "at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)"
    ]
  },
  "nativeStack": [
    {
      "pc": "0x00007ff68df41389",
      "symbol": "std::basic_ostream<char,std::char_traits<char> >::operator<<+10873"
    },
    {
      "pc": "0x00007ff68df457cc",
      "symbol": "std::basic_ostream<char,std::char_traits<char> >::operator<<+28348"
    },
    {
      "pc": "0x00007ff68df44778",
      "symbol": "std::basic_ostream<char,std::char_traits<char> >::operator<<+24168"
    },
    {
      "pc": "0x00007ff68df3dc16",
      "symbol": "std::basic_ostream<char,std::char_traits<char> >::basic_ostream<char,std::char_traits<char> >+5638"
    },
    {
      "pc": "0x00007ff68e7eb150",
      "symbol": "v8::internal::Builtins::builtin_handle+325536"
    },
    {
      "pc": "0x00007ff68e7ea697",
      "symbol": "v8::internal::Builtins::builtin_handle+322791"
    },
    {
      "pc": "0x00007ff68e7ea9d8",
      "symbol": "v8::internal::Builtins::builtin_handle+323624"
    },
    {
      "pc": "0x00007ff68e7ea7de",
      "symbol": "v8::internal::Builtins::builtin_handle+323118"
    },
    {
      "pc": "0x00007ff68ec62d6d",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+568205"
    },
    {
      "pc": "0x00007ff68ebe27fc",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+42524"
    },
    {
      "pc": "0x00007ff68ebdbbac",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+14796"
    },
    {
      "pc": "0x00007ff68ebe27fc",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+42524"
    },
    {
      "pc": "0x00007ff68ebe27fc",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+42524"
    },
    {
      "pc": "0x00007ff68ebe27fc",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+42524"
    },
    {
      "pc": "0x00007ff68ebe27fc",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+42524"
    },
    {
      "pc": "0x00007ff68ebe27fc",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+42524"
    },
    {
      "pc": "0x00007ff68ebe27fc",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+42524"
    },
    {
      "pc": "0x00007ff68ebe27fc",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+42524"
    },
    {
      "pc": "0x00007ff68ebdfdb1",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+31697"
    },
    {
      "pc": "0x00007ff68ebdf99c",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+30652"
    },
    {
      "pc": "0x00007ff68e744f43",
      "symbol": "v8::internal::Execution::CallWasm+1395"
    },
    {
      "pc": "0x00007ff68e7448c6",
      "symbol": "v8::internal::Execution::Call+182"
    },
    {
      "pc": "0x00007ff68e815b3b",
      "symbol": "v8::Function::Call+603"
    },
    {
      "pc": "0x00007ff68e0595ce",
      "symbol": "node::Start+1150"
    },
    {
      "pc": "0x00007ff68e059877",
      "symbol": "node::Start+1831"
    },
    {
      "pc": "0x00007ff68e05874a",
      "symbol": "node::LoadEnvironment+26"
    },
    {
      "pc": "0x00007ff68dffb715",
      "symbol": "EVP_CIPHER_CTX_buf_noconst+30565"
    },
    {
      "pc": "0x00007ff68e059263",
      "symbol": "node::Start+275"
    },
    {
      "pc": "0x00007ff68df1666c",
      "symbol": "RC4_options+339308"
    },
    {
      "pc": "0x00007ff68ed15d58",
      "symbol": "v8::internal::SetupIsolateDelegate::SetupHeap+1301368"
    },
    {
      "pc": "0x00007fff385b7bd4",
      "symbol": "BaseThreadInitThunk+20"
    },
    {
      "pc": "0x00007fff3892ced1",
      "symbol": "RtlUserThreadStart+33"
    }
  ],
  "javascriptHeap": {
    "totalMemory": 58392576,
    "totalCommittedMemory": 58392576,
    "usedMemory": 33373392,
    "availableMemory": 33136984,
    "memoryLimit": 71303168,
    "heapSpaces": {
      "read_only_space": {
        "memorySize": 262144,
        "committedMemory": 262144,
        "capacity": 261872,
        "used": 32296,
        "available": 229576
      },
      "new_space": {
        "memorySize": 33554432,
        "committedMemory": 33554432,
        "capacity": 16759808,
        "used": 9246120,
        "available": 7513688
      },
      "old_space": {
        "memorySize": 16101376,
        "committedMemory": 16101376,
        "capacity": 16060072,
        "used": 16025768,
        "available": 34304
      },
      "code_space": {
        "memorySize": 425984,
        "committedMemory": 425984,
        "capacity": 398816,
        "used": 156288,
        "available": 242528
      },
      "map_space": {
        "memorySize": 266240,
        "committedMemory": 266240,
        "capacity": 264576,
        "used": 181040,
        "available": 83536
      },
      "large_object_space": {
        "memorySize": 3092480,
        "committedMemory": 3092480,
        "capacity": 3091280,
        "used": 3091280,
        "available": 0
      },
      "code_large_object_space": {
        "memorySize": 49152,
        "committedMemory": 49152,
        "capacity": 3552,
        "used": 3552,
        "available": 0
      },
      "new_large_object_space": {
        "memorySize": 4640768,
        "committedMemory": 4640768,
        "capacity": 16759808,
        "used": 4637048,
        "available": 12122760
      }
    }
  },
  "resourceUsage": {
    "userCpuSeconds": 0.375,
    "kernelCpuSeconds": 0.046,
    "cpuConsumptionPercent": 42.1,
    "maxRss": 79261696,
    "pageFaults": {
      "IORequired": 23593,
      "IONotRequired": 0
    },
    "fsActivity": {
      "reads": 1,
      "writes": 5
    }
  },
  "libuv": [
    {
      "type": "async",
      "is_active": true,
      "is_referenced": false,
      "address": "0x00000234a751c6d0"
    },
    {
      "type": "timer",
      "is_active": false,
      "is_referenced": false,
      "address": "0x00000234a8f2f3b8",
      "repeat": 0,
      "firesInMsFromNow": 935839265546,
      "expired": false
    },
    {
      "type": "check",
      "is_active": true,
      "is_referenced": false,
      "address": "0x00000234a8f2f458"
    },
    {
      "type": "idle",
      "is_active": false,
      "is_referenced": true,
      "address": "0x00000234a8f2f4d0"
    },
    {
      "type": "prepare",
      "is_active": false,
      "is_referenced": false,
      "address": "0x00000234a8f2f548"
    },
    {
      "type": "check",
      "is_active": false,
      "is_referenced": false,
      "address": "0x00000234a8f2f5c0"
    },
    {
      "type": "async",
      "is_active": true,
      "is_referenced": false,
      "address": "0x00000234a75aeeb0"
    },
    {
      "type": "async",
      "is_active": true,
      "is_referenced": false,
      "address": "0x00007ff68f91b510"
    },
    {
      "type": "async",
      "is_active": true,
      "is_referenced": false,
      "address": "0x00007ff68f91b610"
    },
    {
      "type": "loop",
      "is_active": false,
      "address": "0x00007ff68fba63e0"
    }
  ],
  "environmentVariables": {
    "": "C:=C:\\Users\\bert\\Documents\\studiohyperdrive\\repos\\nodejs-conf-report\\rtk",
    "": "ExitCode=00000000",
    "FOO": "BAR ",
    "FPS_BROWSER_APP_PROFILE_STRING": "Internet Explorer",
    "CommonProgramFiles": "C:\\Program Files\\Common Files",
    "ProgramFiles": "C:\\Program Files",
    "PROMPT": "$P$G",
    "SESSIONNAME": "Console",
    "COMPUTERNAME": "DESKTOP-K9VNH6V",
    "TERMINAL_EMULATOR": "JetBrains-JediTerm",
    "__INTELLIJ_COMMAND_HISTFILE__": "C:\\Users\\bert\\.IntelliJIdea2019.2\\config\\terminal\\history\\history-454",
    "SystemDrive": "C:",
    "NUMBER_OF_PROCESSORS": "12",
    "LOGONSERVER": "\\\\DESKTOP-K9VNH6V",
    "TEMP": "C:\\Users\\bert\\AppData\\Local\\Temp",
    "GDAL_DATA": "C:\\Program Files\\PostgreSQL\\11\\gdal-data",
    "HOMEPATH": "\\Users\\bert",
    "CI": "true",
    "PATHEXT": ".COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC",
    "": "::=::\\",
    "USERNAME": "bert",
    "ES_HOME": "C:\\Program Files\\Elastic\\Elasticsearch\\6.6.2\\",
    "USERDOMAIN_ROAMINGPROFILE": "DESKTOP-K9VNH6V",
    "CommonProgramFiles(x86)": "C:\\Program Files (x86)\\Common Files",
    "LOCALAPPDATA": "C:\\Users\\bert\\AppData\\Local",
    "PROCESSOR_REVISION": "9e0a",
    "PROCESSOR_IDENTIFIER": "Intel64 Family 6 Model 158 Stepping 10, GenuineIntel",
    "DriverData": "C:\\Windows\\System32\\Drivers\\DriverData",
    "APPDATA": "C:\\Users\\bert\\AppData\\Roaming",
    "ALLUSERSPROFILE": "C:\\ProgramData",
    "USERDOMAIN": "DESKTOP-K9VNH6V",
    "NODE_ENV": "local",
    "NVM_SYMLINK": "C:\\nodejs",
    "PROCESSOR_LEVEL": "6",
    "ProgramData": "C:\\ProgramData",
    "ComSpec": "C:\\WINDOWS\\system32\\cmd.exe",
    "PROCESSOR_ARCHITECTURE": "AMD64",
    "FPS_BROWSER_USER_PROFILE_STRING": "Default",
    "SystemRoot": "C:\\WINDOWS",
    "NVM_HOME": "C:\\Users\\bert\\AppData\\Roaming\\nvm",
    "OneDrive": "C:\\Users\\bert\\OneDrive",
    "PSModulePath": "C:\\Program Files\\WindowsPowerShell\\Modules;C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules",
    "ES_PATH_CONF": "C:\\ProgramData\\Elastic\\Elasticsearch\\config",
    "Path": "C:\\Program Files\\JetBrains\\IntelliJ IDEA 2018.3.4\\jbr\\\\bin;C:\\Program Files\\JetBrains\\IntelliJ IDEA 2018.3.4\\jbr\\\\bin\\server;C:\\Program Files (x86)\\Microsoft SDKs\\Azure\\CLI2\\wbin;C:\\Program Files (x86)\\Common Files\\Oracle\\Java\\javapath;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\iCLS\\;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\iCLS\\;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Windows\\System32\\OpenSSH\\;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\Program Files\\Microsoft VS Code\\bin;C:\\Users\\bert\\AppData\\Roaming\\nvm;C:\\nodejs;C:\\Program Files\\dotnet\\;C:\\Program Files\\Microsoft SQL Server\\Client SDK\\ODBC\\170\\Tools\\Binn\\;C:\\Program Files\\Git\\cmd;C:\\Program Files\\Git\\mingw64\\bin;C:\\Program Files\\Git\\usr\\bin;C:\\Program Files\\PostgreSQL\\11\\bin;C:\\Program Files\\nodejs\\;C:\\Program Files\\Microsoft SQL Server\\Client SDK\\ODBC\\130\\Tools\\Binn\\;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\WINDOWS\\System32\\OpenSSH\\;C:\\Program Files\\Docker\\Docker\\resources\\bin;C:\\ProgramData\\DockerDesktop\\version-bin;C:\\Program Files\\IBM\\Cloud\\bin;C:\\Users\\bert\\AppData\\Local\\Programs\\Python\\Python36\\Scripts\\;C:\\Users\\bert\\AppData\\Local\\Programs\\Python\\Python36\\;C:\\Users\\bert\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Program Files\\JetBrains\\IntelliJ IDEA 2018.3.4\\bin;;C:\\Users\\bert\\AppData\\Roaming\\nvm;C:\\nodejs;C:\\Users\\bert\\AppData\\Roaming\\npm;C:\\Program Files\\hasura;C:\\Users\\bert\\.dotnet\\tools;%USERPROFILE%\\AppData\\Local\\Microsoft\\WindowsApps",
    "ProgramFiles(x86)": "C:\\Program Files (x86)",
    "USERPROFILE": "C:\\Users\\bert",
    "windir": "C:\\WINDOWS",
    "ProgramW6432": "C:\\Program Files",
    "OS": "Windows_NT",
    "TMP": "C:\\Users\\bert\\AppData\\Local\\Temp",
    "PUBLIC": "C:\\Users\\Public",
    "IntelliJ IDEA": "C:\\Program Files\\JetBrains\\IntelliJ IDEA 2018.3.4\\bin;",
    "HOMEDRIVE": "C:",
    "CommonProgramW6432": "C:\\Program Files\\Common Files"
  },
  "sharedObjects": [
    "C:\\Program Files\\nodejs\\node.exe",
    "C:\\WINDOWS\\SYSTEM32\\ntdll.dll",
    "C:\\WINDOWS\\System32\\KERNEL32.DLL",
    "C:\\WINDOWS\\System32\\KERNELBASE.dll",
    "C:\\WINDOWS\\System32\\WS2_32.dll",
    "C:\\WINDOWS\\System32\\RPCRT4.dll",
    "C:\\WINDOWS\\System32\\ADVAPI32.dll",
    "C:\\WINDOWS\\SYSTEM32\\dbghelp.dll",
    "C:\\WINDOWS\\System32\\msvcrt.dll",
    "C:\\WINDOWS\\System32\\ucrtbase.dll",
    "C:\\WINDOWS\\System32\\sechost.dll",
    "C:\\WINDOWS\\System32\\USER32.dll",
    "C:\\WINDOWS\\System32\\win32u.dll",
    "C:\\WINDOWS\\System32\\GDI32.dll",
    "C:\\WINDOWS\\System32\\gdi32full.dll",
    "C:\\WINDOWS\\System32\\msvcp_win.dll",
    "C:\\WINDOWS\\System32\\PSAPI.DLL",
    "C:\\WINDOWS\\System32\\CRYPT32.dll",
    "C:\\WINDOWS\\SYSTEM32\\IPHLPAPI.DLL",
    "C:\\WINDOWS\\SYSTEM32\\USERENV.dll",
    "C:\\WINDOWS\\System32\\MSASN1.dll",
    "C:\\WINDOWS\\System32\\bcrypt.dll",
    "C:\\WINDOWS\\System32\\profapi.dll",
    "C:\\WINDOWS\\SYSTEM32\\WINMM.dll",
    "C:\\WINDOWS\\SYSTEM32\\winmmbase.dll",
    "C:\\WINDOWS\\System32\\cfgmgr32.dll",
    "C:\\WINDOWS\\System32\\bcryptPrimitives.dll",
    "C:\\WINDOWS\\System32\\IMM32.DLL",
    "C:\\WINDOWS\\System32\\powrprof.dll",
    "C:\\WINDOWS\\System32\\UMPDC.dll",
    "C:\\WINDOWS\\system32\\uxtheme.dll",
    "C:\\WINDOWS\\System32\\combase.dll",
    "C:\\WINDOWS\\system32\\mswsock.dll",
    "C:\\WINDOWS\\System32\\kernel.appcore.dll",
    "C:\\WINDOWS\\System32\\NSI.dll",
    "C:\\WINDOWS\\SYSTEM32\\dhcpcsvc6.DLL",
    "C:\\WINDOWS\\SYSTEM32\\dhcpcsvc.DLL",
    "C:\\WINDOWS\\SYSTEM32\\DNSAPI.dll",
    "C:\\WINDOWS\\system32\\napinsp.dll",
    "C:\\WINDOWS\\system32\\pnrpnsp.dll",
    "C:\\WINDOWS\\System32\\winrnr.dll",
    "C:\\WINDOWS\\system32\\NLAapi.dll",
    "C:\\WINDOWS\\system32\\wshbth.dll"
  ]
}
```

The inspect command: 
```
rtk inspect report.20191113.151429.7984.0.001.json
```

The output of the inspect command:
```
C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\rxjs\internal\util\hostReportError.js:4
    setTimeout(function () { throw err; }, 0);
                             ^

RTkError: Encountered a thing that does not look like a report!
    at create (C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\@report-toolkit\common\dist\report-toolkit-common.cjs.js:222:12)
    at throwRTkError (C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\@report-toolkit\common\dist\report-toolkit-common.cjs.js:402:69)
    at C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\@report-toolkit\inspector\dist\report-toolkit-inspector.cjs.js:935:54
    at C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\rxjs\internal\util\pipe.js:20:56
    at Array.reduce (<anonymous>)
    at piped (C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\rxjs\internal\util\pipe.js:20:20)
    at Observable.pipe (C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\rxjs\internal\Observable.js:92:48)
    at fromFilepathsToReports (C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\@report-toolkit\cli\dist\report-toolkit-cli.cjs.js:156:29)
    at Object.handler$1 [as handler] (C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\@report-toolkit\cli\dist\report-toolkit-cli.cjs.js:377:26)
    at C:\Users\bert\AppData\Roaming\npm\node_modules\report-toolkit\node_modules\yargs\lib\command.js:241:49 {
  code: 'RTKERR_INVALID_REPORT',
  data: undefined,
  url: undefined
}
```

Environment:
Windows 10: 1903 (OS Build 18362.418)
Node.js: v12.13.0
report-toolkit version: 0.2.0

**Comment by boneskull** [comment]
I've confirmed this issue

**Comment by boneskull** [comment]
This is what happens when I don't test on Windows.  Reports generated on Windows do _not_ have a `userLimits` property.

**Comment by boneskull** [comment]
@bertyhell Do you mind if I pull this report file into the project as a test fixture?

**Comment by boneskull** [comment]
This has been fixed in v0.2.3.

**Comment by bertyhell** [comment]
>  Do you mind if I pull this report file into the project as a test fixture?

No, go ahead
